pkg/cmd/server/start/start_master.go

```diff
@@ -498,14 +491,15 @@ func StartAPI(oc *origin.MasterConfig, kc *kubernetes.MasterConfig) error {
 		}
 
 		if oc.Options.AssetConfig.ServingInfo.BindAddress == oc.Options.ServingInfo.BindAddress {
-			unprotectedInstallers = append(unprotectedInstallers, config)
+			embeddedAssetConfig = config
+			proxyAssetConfig = config
```

Comment:
why distinguish between embeddedAssetConfig and proxyAssetConfig? we're not proxying to the asset server